test/vlq-tests.ts

```diff
@@ -109,29 +109,27 @@ describe('test encode', function() {
   it('encoder', function() {
     // (lines + segemnts * 6) = byte_count
     let decoded = {
-      mappings: {
```

Comment:
Is this white space only? If this is linter related can it be a separate PR? This PR is already not very focused